### PR TITLE
Created action type constants and action creators for organizations

### PR DESCRIPTION
**Pull request by jdspell**
# Description

Created action type constants and action creators for basic CRUD functionality for organizations.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Change status
- [ ] Complete, tested, ready to review and merge
- [x] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] My code has been reviewed by at least one peer
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] There are no merge conflicts

### CODE REVIEW

**Review by JorPoon — DISMISSED** [review]
is it good to console.log(res) and console.log(error) ?

File: src/actions/organizationActions.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+        .catch(err => {
+            dispatch({ type: CREATE_ORGANIZATION_FAILURE, payload: err })
+        });
+};
+
+export const updateOrganization = (id, updatedOrg) => dispatch => {
+    dispatch({ type: UPDATE_ORGANIZATION_START });
+    
+    return axios    
+        .put(`${url}orgs/${id}`, updatedOrg)
```
Comment (JorPoon) [inline src/actions/organizationActions.js]: .put(`${url}/orgs/${id}`, updatedOrg)

**Comment by Estrax** [comment]
I would suggest against. You don't want client to see a lot of logs in their console. May be acceptable for development, but not for the production app.

**Comment by Estrax** [comment]
So if you want them for development purposes, keep adding them, but before the final release I will ask you to remove all of them :)

**Review by Estrax — DISMISSED** [review]

**Review by JorPoon — DISMISSED** [review]
Looks good

**Review by tfolbrecht — DISMISSED** [review]
Setting an env var for the url const would make life easier for flipping between localhost, production, etc

File: src/pages/UserLogin.js [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+      this.props.loginUser(this.state);
+      //add .then that redirects to the user's page
+  }
+
+  render(){
+    return(
+        <div className="UserLogin">
+            <Form onSubmit={this.handleSubmit}>
+                <FormGroup>
+                    <Label for="username"></Label>
```
Comment (Estrax) [inline src/pages/UserLogin.js:39]: use `<Label for="username" />` instead

**Review by Estrax — APPROVED** [review]
Looks okay overall.

File: src/pages/UserLogin.js (hunk 2 above)
Comment (jdspell) [reply src/pages/UserLogin.js:39]: ok i will  adjust that

**Review by tfolbrecht — APPROVED** [review]

File: src/reducers/organizations.js [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+
+        case UPDATE_ORGANIZATION_START:
+        return {
+            ...state,
+        }
+
+        case UPDATE_ORGANIZATION_SUCCESS:
+        return {
+            ...state,
+            organization: action.payload
```
Comment (JorPoon) [inline src/reducers/organizations.js:70]: if organization in state is an array of organizations ? should it be [...state.organization, action.payload] ?

**Review by JorPoon — APPROVED** [review]

**Pull request merged**